zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -848,7 +831,7 @@ private int durationIndexBucket(long ts) {
                 .setString("service_name", serviceName)
                 .setString("span_name", spanName)
                 .setInt("bucket", durationIndexBucket(timestamp))
-                .setLong("ts", timestamp)
+                .setBytesUnsafe("ts", serializeTs(timestamp))
```

Comment:
I prefer we leave the serialization hack for now, as there's a comment
where this goes away in datastax 3 (as you can use long directly)